packages/CollabCodeEdit-Core.package/CCEEdit.class/class/newWith.in..st

```diff
@@ -0,0 +1,5 @@
+instance creation
+newWith: anObject in: aClass
```

Comment:
Input unspecified, please rename (aMethodSource?)

Reply:
This is intentionally `anObject`, as the type of `editContent` is not specified by `CCEEdit`, but rather by it's subclasses

Reply:
Then i would go for the first all possibilities covering class, which would be `CCEEdit` right?

Reply:
No it is not. The `editcontent` is a variable in a `CCEEdit`, therefore it should never be a `CCEEdit`.